certif/tests/acceptance/session-add-candidate_test.js

```diff
@@ -0,0 +1,69 @@
+import { visit as visitScreen } from '@1024pix/ember-testing-library';
+import { click, fillIn } from '@ember/test-helpers';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import { setupApplicationTest } from 'ember-qunit';
+import { module, test } from 'qunit';
+
+import { authenticateSession } from '../helpers/test-init';
+
+module('Acceptance | Session Add Candidate', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  module('When certification center is V3 pilot for pix/pix+ split', function () {
```

Comment:
Allez, moi aussi j'en propose une :D

```suggestion
  module('When certification center is pix/pix+ split pilot and V3 pilot', function () {
```